docs/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thank you for your pull request!
+
+Please see https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md for guidelines on
+how to best contribute to the Windows App SDK Samples!
```

Comment:
>to the Windows App SDK Samples [](http://example.com/codeflow?start=23&length=30)

Grammar-wise, I think we should use "to the Windows App SDK Samples project" or "to Windows App SDK Samples". I think the former is better (since it's explicit).

Reply:
Windows App SDK Samples repository?